scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
This is going to need some more thought; I suspect running ./release on multiple machines is going to lead to a lot of headaches w/r/t reproducible build hashes for signing.

Login is a no-go; the release process already rather tedious.

/cc @xiang90